cookbook/security/entity_provider.rst

```diff
@@ -5,69 +5,49 @@
 How to Load Security Users from the Database (the Entity Provider)
 ==================================================================
 
-The security layer is one of the smartest tools of Symfony. It handles two
-things: the authentication and the authorization processes. Although it may
-seem difficult to understand how it works internally, the security system
-is very flexible and allows you to integrate your application with any authentication
-backend, like Active Directory, an OAuth server or a database.
+Symfony's security system can load security users from anywhere - like a
+database, via Active Directory or an OAuth server. This article will show
+you how to load your users from the database via a Doctrine entity.
 
 Introduction
 ------------
 
-This article focuses on how to authenticate users against a database table
-managed by a Doctrine entity class. The content of this cookbook entry is split
-in three parts. The first part is about designing a Doctrine ``User`` entity
-class and making it usable in the security layer of Symfony. The second part
-describes how to easily authenticate a user with the Doctrine
-:class:`Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider` object
-bundled with the framework and some configuration.
-Finally, the tutorial will demonstrate how to create a custom
-:class:`Symfony\\Bridge\\Doctrine\\Security\\User\\EntityUserProvider` object to
-retrieve users from a database with custom conditions.
+Before you start, you should check out the `FOSUserBundle`_. This external
```

Comment:
is it really worth adding this section as the main (and first) paragraph? FOSUserBundle is great, but not for some more complex things. And to get to know things, it might be even better to actually create a User Provider.